src/Data/List/Relation/Binary/Sublist/Heterogeneous/Properties.agda

```diff
@@ -400,26 +400,22 @@ module Antisymmetry
 
   open ℕ.≤-Reasoning
 
+  private
+    antisym-lemma : ∀ xs ys₁ y → Sublist R xs ys₁ → ¬ Sublist S (y ∷ ys₁) xs
+    antisym-lemma xs ys₁ y rs ss = ℕ.<-irrefl ≡.refl (begin
+      length (y ∷ ys₁) ≤⟨ length-mono-≤ ss ⟩
+      length xs        ≤⟨ length-mono-≤ rs ⟩
+      length ys₁       ∎)
```

Comment:
Conservatively, I made this lemma private.
There could potentially be some use of this lemma in other contexts, but then it would need a proper name.